### PR TITLE
Tag Libz.jl v0.2.2

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/Libz.jl](https://github.com/BioJulia/Libz.jl)
Release: [v0.2.2](https://github.com/BioJulia/Libz.jl/releases/tag/v0.2.2)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/Libz.jl/builds/190674992)
Diff: [vs v0.2.1](https://github.com/BioJulia/Libz.jl/compare/0209f706f92f5a90d117bf0afc089fb018190661...e0d26595f232a4267dc78a430462033f75e806bd)
`requires` vs v0.2.1: 
```diff
--- 0.2.1/requires
+++ 0.2.2/requires
@@ -1,3 +1,3 @@
 julia 0.4
 Compat 0.8.0
-BufferedStreams 0.2
+BufferedStreams 0.2.1
```
cc: @Ward9250

### CODE REVIEW

File: Libz/versions/0.2.2/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Compat 0.8.0
```
Comment (tkelman) [inline Libz/versions/0.2.2/requires]: `take!` needs 0.9.5

**Comment by tkelman** [comment]
will modifying the order of arguments break anyone who might have been using unexported functions here?

**Comment by KristofferC** [comment]
Curious, why would that matter?

**Comment by tkelman** [comment]
which part? breaking people who've been using unexported functions? this is a pretty widely used package, and until we have auto-pkgeval running reverse dependency tests it would be good to at least think about whether or not this is likely to break anything (even if it's only people who've been doing things they shouldn't be, but as we know that can be an unfortunately large slice of the julia ecosystem)

File: Libz/versions/0.2.2/requires (hunk 1 above)
Comment (Ward9250) [inline Libz/versions/0.2.2/requires]: To be clear, Compat 0.9.5?

**Comment by Ward9250** [comment]
Ok, close this, I'm going to get rid of this release and tag and do it again and cherry-pick it onto the release-0.2 branch of Libz.

**Comment by tkelman** [comment]
will you re-tag as the same version number? if so, attobot will replace the content of this branch in place and we should leave it open (but not merge yet) to avoid a github bug

**Comment by Ward9250** [comment]
@tkelman Yes I did it so as it's the same tag number.

**Comment by musm** [comment]
@Ward9250 thanks for the speedy work!

**Comment by Ward9250** [comment]
No problem! First time I retroactively applied bug fixes made to master to a release branch! Read quite a bit re cherry pick vs rebase.

**Pull request merged**